app/insomnia/dynamic-variables.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Dynamic variables
+
+description: Define variables dynamically using scripts and iteration data.
+
+content_type: reference
+layout: reference
+breadcrumbs: 
+  - /insomnia/
+search_aliases:
+  - iteration data
+  - transient variables
+products:
+  - insomnia
+
+related_resources:
+  - text: Requests
+    url: /insomnia/requests/
+  - text: Collections
+    url: /insomnia/collections/
+  - text: Scripts
+    url: /insomnia/scripts/
+  - text: Environments
+    url: /insomnia/environments/
+  - text: Template tags
+    url: /insomnia/template-tags/
+---
+
+Besides [environments](/insomnia/environments/), there are two other ways to define variables in Insomnia:
```

Comment:
```suggestion
You can use variables in Insomnia to do things like reuse environment settings or use configuration from a request. Besides [environments](/insomnia/environments/), there are two other ways to define variables in Insomnia:
```
I completely made that first sentence up, feel free to modify it. I wanted an intro sentence like that that tells me what I can do with variables in Insomnia.